tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -320,6 +320,7 @@
   "DD_TRACE_GIT_METADATA_ENABLED": "git_metadata_enabled",
   "DD_TRACE_ENABLED": "trace_enabled",
   "DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED": "experimental_appsec_standalone_enabled",
```

Comment:
do we want to keep the older env var here?

```suggestion
```

Reply:
It's still in the go json, so...

Reply:
Yes, we always need to keep variables there after they've been added, because old libraries will continue to send the old value

Reply:
only add, never remove